### PR TITLE
PYIC-2865 Modify FE for -Prove your identity another way- link to 'journey/claimIdentity'

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
Modify FE for -Prove your identity another way- link to 'journey/claimIdentity'
### What changed
Modified FE for -Prove your identity another way- link to 'journey/claimIdentity'

### Issue tracking

- [PYIC-2865](https://govukverify.atlassian.net/browse/PYIC-2865)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-2865]: https://govukverify.atlassian.net/browse/PYIC-2865?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by AmritSidhu — DISMISSED** [review]

**Review by chris-dco — CHANGES_REQUESTED** [review]
@rpayal is it possible to change `claimIdentity` to `claimedIdentity`? I am happy to approve then.

File: src/views/ipv/page-multiple-doc-check.njk [hunk 1]
```diff
@@ -33,7 +33,7 @@
                 divider: 'pages.pageMultipleDocCheck.content.formRadioButtons.separateOptionsInFormText' | translate
               },
               {
-                value: "end",
+                value: "next/claim-identity",
```
Comment (chris-dco) [inline src/views/ipv/page-multiple-doc-check.njk]: Should we do this on a feature flag? This is due to a comment on https://govukverify.atlassian.net/browse/PYIC-2834 where it says ‘How do we make this only available when the F2F is enabled???’

Comment (chris-dco) [reply src/views/ipv/page-multiple-doc-check.njk]: Also can you make it just "claimedIdentity".

File: src/views/ipv/page-multiple-doc-check.njk [hunk 2]
```diff
@@ -33,7 +33,7 @@
                 divider: 'pages.pageMultipleDocCheck.content.formRadioButtons.separateOptionsInFormText' | translate
               },
               {
-                value: "end",
+                value: "next/claimed-identity",
```
Comment (chris-dco) [inline src/views/ipv/page-multiple-doc-check.njk:36]: Also can you make it just "claimedIdentity" and feature flagged as this is only available in dev at the moment.

Comment (chris-dco) [inline src/views/ipv/page-multiple-doc-check.njk:36]: Should we do this on a feature flag? This is due to a comment on https://govukverify.atlassian.net/browse/PYIC-2834 where it says ‘How do we make this only available when the F2F is enabled???’

**Review by AmritSidhu — APPROVED** [review]

**Comment by kerrr** [comment]
I don't think we need this and it can be closed.

**Comment by rpayal** [comment]
Not needed any more as done using state machine events. (so no FE change needed here)

**Pull request closed without merging**